docs/contributing/development-policies.rst

```diff
@@ -0,0 +1,219 @@
+.. _development_policies:
+
+####################
+Development policies
+####################
+
+.. _reporting_security_issues:
+
+*************************
+Reporting security issues
+*************************
+
+.. ATTENTION::
+
+    If you think you have discovered a security issue in our code, please report
+    it **privately**, by emailing us at `security@django-cms.org`_.
+
+        Please **do not** raise it on:
+
+        * IRC
+        * GitHub
+        * either of our email lists
+
+        or in any other public forum until we have had a chance to deal with it.
+
+
+******
+Review
+******
+
+All patches should be made as pull requests to `the GitHub repository <https://github.com/divio/django-cms>`_. Patches
+should never be pushed directly.
+
+**Nothing** may enter the code-base, *including the documentation*, without proper review and formal approval from the
+core team.
+
+Reviews are welcomed by all members of the community. You don't need to be a core developer, or even an experienced
+programmer, to contribute usefully to code review. Even noting that you don't understand something in a pull request
+is valuable feedback and will be taken seriously.
+
+
+Formal approval
+===============
+
+Formal approval means "OK to merge" comments, following review, from at least two different members of the core team
+who have expertise in the relevant areas, and excluding the author of the pull request.
+
+The exceptions to this are frontend code and documentation, where one "OK to merge" comment will suffice, at least
+until the team has more expert developers in those areas.
+
+
+**********************************************
+Proposal and discussion of significant changes
+**********************************************
+
+New features and backward-incompatible changes should be proposed using the `django CMS developers email list
+<http://groups.google.com/group/django-cms-developers>`_. Discussion should take place there before any pull requests
+are made.
+
+This is in the interests of openness and transparency, and to give the community a chance to participate in and
+understand the decisions taken by the project.
+
+
+****************
+Release schedule
+****************
+
+The `roadmap <https://github.com/divio/django-cms/wiki/Roadmap>`_ can be found on our GitHub wiki
+page.
+
+We are planning releases according to **key principles and aims**. Issues within milestones are
+therefore subject to change.
+
+The `django CMS developers email list <http://groups.google.com/group/django-cms-developers>`_ serves as gathering
+point for developers. We submit ideas and proposals prior to the roadmap goals.
```

Comment:
@evildmp do we somewhere also describe that the roadmap is set by the technical board according to feedback in the developers group and the core developers? Eventually a reference here or additional content.